internal_transfers/actions/tests/e2e/getInternalImbalance.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { getInternalImbalance, MinimalTxData } from "../../src/accounting";
+import { TenderlySimulator } from "../../src/simulate/tenderly";
+import { ethers } from "ethers";
+
+const simulator = new TenderlySimulator(
+  process.env["TENDERLY_USER"] || "INVALID_USER",
```

Comment:
One alternative to this way of `process.env` handling is that;

* At the start of the application, have a check that runs and throws if anything from env vars is not present
* Add a`module augmentation` of `process.env` for typescript to expect said variable.

Which would allow you to write `process.env.TENDERLY_USER` for example.

A `module augmentation` would be like the following;

```
declare global {
  namespace NodeJS {
    interface ProcessEnv {
      TENDERLY_USER: string;
      TENDERLY_PROJECT: string;
      TENDERLY_ACCESS_KEY: string;
    }
  }
}

```

Reply:
Alternatively you can use the package https://www.npmjs.com/package/ts-dotenv

Reply:
Actually I don't want this to error here because the real instance is only used for the skipped tests and is not expected to be run as part of CI. However, I am glad I finally got a repo as on how to handle env vars. The production code won't actually access env vars like this - only in the e2e tests.